packages/react/src/footer/Footer.tsx

```diff
@@ -17,10 +17,8 @@ export const Footer = ({ background = 'body', children }: FooterProps) => {
 			paddingY={3}
 			borderTop
 			borderTopWidth="xl"
+			borderColor="accent"
 			css={{
-				borderColor: boxPalette.accent,
-				[localPaletteVars.border]: boxPalette.accent,
```

Comment:
In the `Footer` component, we only ever set the border colour to `accent` so there is no need for this `localPaletteVars`. I think this is leftover from when footer was a little more complicated and had different border colours.